File: pruebas_emod/Login/EMOD-212.py
import unittest
from selenium import webdriver
from selenium.webdriver.common.keys import Keys

class PythonOrgSearch(unittest.TestCase):

    def setUp(self):
        self.driver = webdriver.Firefox()

    def test_search_in_python_org(self):
        driver = self.driver
        driver.get("http://emod-frontend-qa.internal.tidnode.cl")
        inputId = driver.find_element_by_css_selector("#userid")
        inputId.send_keys("test")
        inputPass = driver.find_element_by_css_selector("#password")
        inputPass.send_keys("noValida")
        loginBtn = driver.find_element_by_css_selector(".btn.btn-primary")
        loginBtn.click()
        assert "Combinación usuario/contraseña incorrecta" in driver.page_source
        

    def tearDown(self):
        self.driver.close()

if __name__ == "__main__":
    unittest.main()

File: pruebas_emod/Pruebas en Casperjs/login/EMOD-212.js
casper.test.begin('Iniciar sesion con usuario valido contraseña incorrecta', 2, function suite(test){
	casper.start('http://emod-frontend-qa.internal.tidnode.cl/', function(){
		casper.capture('screenshots/212/login1.png');
		test.assertTitle("EMOD - tidchile", "Emod homepage title is the one expected");
		this.sendKeys('#userid', "test"); //usuario valido
		this.sendKeys('#password', "1"); //contraseña incorrecta
		casper.capture('screenshots/212/login2.png');
		this.click('button.btn.btn-primary');		
	});
	casper.then(function() {
		casper.waitForSelector('div.alert.alert-danger', function(){
			casper.capture('screenshots/212/login3.png');
			test.assertSelectorHasText('div.alert.alert-danger','Combinación usuario/contraseña incorrecta');
		});
    });

	casper.run(function(){
		test.done();
	});
});